test/Npgsql.Tests/CopyTests.cs

```diff
@@ -359,7 +359,7 @@ public void UnexpectedExceptionBinaryImport()
             }
         }
 
-        [Test, IssueLink("https://github.com/npgsql/npgsql/issues/657")]
+        //[Test, IssueLink("https://github.com/npgsql/npgsql/issues/657")]
```

Comment:
Why is it commented?

Reply:
Oops, thanks for catching this. [Resharper and Rider currently have a bug](https://youtrack.jetbrains.com/issue/RIDER-11989) where they don't respect `[Explicit]` and always run the test. For this kind of heavy test it makes it impossible to run tests as part of development. Remove the comment.